### PR TITLE
Build tests with ENB

### LINKED ISSUE #478: Build tests with ENB

### PR DESCRIPTION
**Pull request by blond**
Resolved #478 

Run:

``` bash
$ ./node_modules/.bin/enb make sets
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +43,7 @@
   "private": true,
   "scripts": {
     "deps": "bower-npm-install --non-interactive",
-    "build": "enb make docs -n",
+    "build": "enb make sets && enb make docs && enb make",
     "test": "npm run lint && npm run unit",
     "lint": "jshint-groups && jscs . && csscomb -vl .",
     "unit": "bower-npm-install && bem make -w 10 desktop.specs touch-pad.specs touch-phone.specs",
```
Comment (arikon) [inline package.json:50]: @andrewblond Should fix here also

File: package.json [hunk 2]
```diff
@@ -42,7 +43,7 @@
   "private": true,
   "scripts": {
     "deps": "bower-npm-install --non-interactive",
-    "build": "enb make docs -n",
+    "build": "enb make sets && enb make docs && enb make",
```
Comment (arikon) [inline package.json]: @andrewblond `enb make` does not build `docs` and `sets` by default?

File: package.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
     "istanbul": "^0.2.7",
     "enb": "0.10.0",
     "enb-bem-docs": "git://github.com/andrewblond/enb-bem-docs.git#3cd156684d92af632d9cf6337b023fa2c29c0587",
+    "enb-bem-sets": "git://github.com/andrewblond/enb-bem-sets.git#d4aba0185d4006cd26f756730e44a807a7b917df",
```
Comment (arikon) [inline package.json]: @andrewblond Why use git hashes here and not versions?

File: package.json (hunk 1 above)
Comment (narqo) [inline package.json:50]: @arikon as far as I understand this PR is about building docs and test-pages. Unit tests (`*.specs`) are not included here.

File: package.json (hunk 2 above)
Comment (blond) [inline package.json]: `docs` и `sets` это таски, а таски в `enb` не запускаются по `enb make`

File: package.json (hunk 1 above)
Comment (blond) [inline package.json:50]: ПР про сборку примеров для разработчиков, которые в `bem-pr` называются `tests`.

**Comment by blond** [comment]
Updated

**Pull request merged**